bin/pulsar-perf

```diff
@@ -137,6 +137,14 @@ if [ "$COMMAND" == "produce" ]; then
     exec $JAVA $OPTS com.yahoo.pulsar.testclient.PerformanceProducer --conf-file $PULSAR_PERFTEST_CONF "$@"
 elif [ "$COMMAND" == "consume" ]; then
     exec $JAVA $OPTS com.yahoo.pulsar.testclient.PerformanceConsumer --conf-file $PULSAR_PERFTEST_CONF "$@"
+elif [ "$COMMAND" == "monitor" ]; then
+    exec $JAVA $OPTS com.yahoo.pulsar.testclient.BrokerMonitor "$@"
```

Comment:
The difference is that one monitors data when `SimpleLoadManagerImpl` is used and the other monitors data when `ModularLoadManagerImpl` is used. I have renamed them to `SimpleLoadManagerBrokerMonitor` and `ModularLoadManagerBrokerMonitor` to better reflect the difference.